templates/security/index.html

```diff
@@ -6,337 +6,388 @@
+                A new LTS (Long Term Support) version of Ubuntu is released every two years, for desktop and server. Both versions receive updates and are supported for ten years.
+              </p>
+            </div>
+            <div class="col-3 col-medium-2">
+              <h3 class="p-heading--5">Expanded security</h3>
+              <p>
+                Canonical offers Expanded Security Maintenance (ESM) for infrastructure and applications to provide kernel livepatches and vulnerability fixes through a secure and private archive.
+              </p>
+            </div>
+            <div class="col-3 col-medium-2">
+              <h3 class="p-heading--5">FIPS</h3>
+              <p>
+                Ubuntu provides you with FIPS 140 certified cryptographic packages enabling Linux workloads to run on U.S. government regulated and high security environments.
+              </p>
+            </div>
           </div>
-        </li>
-        <li class="p-matrix__item">
-          <div>
-            <h3 class="p-matrix__title p-heading--4">Secure snap packages</h3>
-            <p class="p-matrix__desc">
-              Software packages delivered as strict-mode snaps are fully confined using AppArmor, device cgroups, and seccomp.
-            </p>
+        </div>
+        <hr class="p-rule--muted u-hide--small" />
+        <div class="p-section--shallow">
+          <div class="row">
+            <div class="col-3 col-medium-2">
+              <h3 class="p-heading--5">Designed to be secure</h3>
+              <p>
+                Linux is based on Unix. It inherits Discretionary Access Control and includes Mandatory Access Control via AppArmor.
+              </p>
+            </div>
+            <div class="col-3 col-medium-2">
+              <h3 class="p-heading--5">Protected VMs</h3>
+              <p>
+                LXD containers, libvirt VMs and OpenStack VMs are protected by AppArmor by default. A rich set of profiles are provided so users can opt-in to protection for other applications.
+              </p>
+            </div>
+            <div class="col-3 col-medium-2">
+              <h3 class="p-heading--5">Secure snap packages</h3>
+              <p>Software packages delivered as strict-mode snaps are fully confined using AppArmor, device cgroups, and seccomp.</p>
+            </div>
           </div>
-        </li>
-      </ul>
+        </div>
+      </div>
     </div>
   </section>
 
-  {% include "shared/_case-study-itstrategen.html" %}
+  <section class="p-section" id="stories">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Learn how ITstrategen keeps their applications secure with Ubuntu</h2>
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
+          <div class="p-image-container--16-9 is-highlighted is-cover">
+            {{ image(url="https://assets.ubuntu.com/v1/d2c94444-it-strategen.png",
+                        alt="",
```

Comment:
```suggestion
                        alt="ITstrategen logo",
```

Reply:
Generally I would not put "logo" but I guess here it would make it clear.

Reply:
This is a decorative image so the missing alt text is intentional

Reply:
oh, I thought it was a logo for the company